lib/events/dynamoevents/dynamoevents.go

```diff
@@ -392,6 +409,16 @@ func (l *Log) handleAWSValidationError(ctx context.Context, err error, sessionID
 	return nil
 }
 
+func (l *Log) handleConditionError(ctx context.Context, sessionID string, in apievents.AuditEvent) error {
+	in.SetIndex(l.Clock.Now().UnixNano())
+
+	if err := l.putAuditEvent(ctx, sessionID, in); err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
nit: is it possible to handle when `isAWSValidationError` and `trace.IsAlreadyExists(convertError(err))` conditions are true?